tests/v3_api/test_import_cluster.py

```diff
@@ -0,0 +1,131 @@
+import os
+import requests
+from lib.aws import AmazonWebServices
+from .common import *  # NOQA
+RANCHER_CLEANUP_CLUSTER = os.environ.get('RANCHER_CLEANUP_CLUSTER', "True")
+
+DATA_SUBDIR = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                           'resource')
+
+
+def test_import_rke_cluster():
+
+    client = get_admin_client()
+
+    # Create nodes in AWS
+    aws_nodes = create_nodes()
+    clusterfilepath = create_rke_cluster_config(aws_nodes)
+
+    # Create RKE K8s Cluster
+    execute_rke_cmd("up", clusterfilepath)
+
+    # Get the cluster Id
```

Comment:
@soumyalj You should be able to use create calls for cluster and cluster_registration_token to create imported clusters.